Ruby/Ruby.sublime-syntax

```diff
@@ -272,11 +275,11 @@ contexts:
     # floating point numbers
     - match: |-
         \b(?x:
-          {{digit}} (?:
+          {{ddigits}} (?:
             # 1e1, 1.1e1
-            (?: (\.)     {{digit}}  )?  {{exponent}} \b
+            (?: (\.)     {{ddigits}}  )?  {{exponent}} \b
```

Comment:
Ouff. Just see, I missed to remove the whitespace as in the pattern above. One space as separator should be enough.

Reply:
Thanks, fixed.